integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -4709,21 +4715,27 @@ msgstr "Mehrfachaktion auswählen"
 msgid "Archive events"
 msgstr "Veranstaltungen archivieren"
 
-#: cms/templates/events/event_list.html
-msgid "Automatically translate this event into Easy German"
-msgstr "Diese Veranstaltung automatisch in Leichte Sprache übersetzen"
-
-#: cms/templates/events/event_list.html cms/templates/pois/poi_list.html
-msgid "Create missing translations automatically"
-msgstr "Fehlende Übersetzungen automatisch erstellen"
+#: cms/templates/events/event_list.html cms/templates/pages/page_tree.html
+#: cms/templates/pois/poi_list.html
+#, python-format
+msgid "Machine translate %(content_type)s via %(provider)s to %(language)s"
+msgstr ""
+"%(content_type)s maschinell via %(provider)s auf %(language)s übersetzen"
```

Comment:
This is quite picky, and maybe not even possible, but as far as I know it would be the correct term to say "ins {language}e übersetzen", like "ins Englische übersetzen". I went through all languages we have in our live system and that are supported by DeepL and checked that it would work that way, and it does. 
So maybe we should do it, but also maybe it's better to ask someone from the service-team which wording they'd prefer